cmd/influx/main.go

```diff
@@ -37,17 +38,38 @@ func main() {
 	fs := flag.NewFlagSet("default", flag.ExitOnError)
 	fs.StringVar(&c.Host, "host", default_host, "influxdb host to connect to")
 	fs.IntVar(&c.Port, "port", default_port, "influxdb port to connect to")
-	fs.StringVar(&c.Username, "username", c.Username, "username to connect to the server.  can be blank if authorization is not required")
-	fs.StringVar(&c.Password, "password", c.Password, "password to connect to the server.  can be blank if authorization is not required")
+	fs.StringVar(&c.Username, "username", c.Username, "username to connect to the server.")
+	fs.StringVar(&c.Password, "password", c.Password, `password to connect to the server.  Leaving blank will prompt for password (--password="")`)
 	fs.StringVar(&c.Database, "database", c.Database, "database to connect to the server.")
 	fs.Parse(os.Args[1:])
 
+	var promptForPassword bool
+	// determine if they set the password flag but provided no value
+	for _, v := range os.Args {
+		v = strings.Replace(v, "-", "", -1)
```

Comment:
Not checking the password being set was an oversight, good catch.  I also made the arg check tighter and only accept `-password` or `--password`, even though `password` is valid based on flag arguments to go.